bin/zopen-init

```diff
@@ -222,10 +214,63 @@ init()
       fi
     fi
   fi
+}
+
+determineStatsCollection()
+{
+  if isIBMHostname; then
+    isHostIBM=true
+  fi
+  if [ -n "$enableStats" ] && $enableStats; then
+    isCollectingStats=true
+  elif [ -n "$enableStats" ] && ! $enableStats; then
+    isCollectingStats=false
+  elif $isHostIBM; then
+    printAttention "Attention: Turning on usage statistics as your host is an IBM host. If this is incorrect, rerun with --noenable-stats"
+    isCollectingStats=true
+  else
+    printInfo "Do you want to contribute anonymous usage statistics to help improve z/OS Open Tools? (press y to collect stats or n for no): "
+    read collectStats < /dev/tty
+    if [ "y" = "${collectStats}" ]; then
+      isCollectingStats=true
+    fi
+  fi
+
+  # Undocumented envar, for CI/CD tracking
+  if [ -n "${ZOPEN_IS_BOT}" ]; then
+    isBot=true
```

Comment:
any reason not to document it?

Reply:
I wanted it to be exclusively by our CI/CD setup. If we expose it and document it, there could be a chance that can be used outside of our CI/CD.

Reply:
ok. it's exposed and not documented, but i am ok with that.